test/unit/sample_files/write_init_files/physics_inputs_4D.F90

```diff
@@ -130,10 +142,35 @@ subroutine physics_read_data(file, suite_names, timestep, read_initialized_varia
                   ! If an index was found in the constituent hash table, then read in the data
                   ! to that index of the constituent array
 
+                  var_found = .false.
                   field_data_ptr => cam_constituents_array()
                   call read_field(file, ccpp_required_data(req_idx),                              &
                        [ccpp_required_data(req_idx)], 'lev', timestep,                            &
-                       field_data_ptr(:,:,constituent_idx), mark_as_read=.false.)
+                       field_data_ptr(:,:,constituent_idx), mark_as_read=.false.,                 &
+                       error_on_not_found=.false., var_found=var_found)
+                  if(.not. var_found) then
+                     const_props => cam_model_const_properties()
+                     constituent_has_default = .false.
+                     call const_props(constituent_idx)%has_default(constituent_has_default,       &
+                          constituent_errflg, constituent_errmsg)
+                     if (constituent_has_default) then
+                        call                                                                      &
+                             const_props(constituent_idx)%default_value(constituent_default_value, constituent_errflg,&
+                             constituent_errmsg)
```

Comment:
Sure!  Would this be the preferred spacing?
```fortran
call const_props(constituent_idx)%                                                       &
         default_value(constituent_default_value, constituent_errflg, constituent_errmsg)
```

Reply:
I think I would probably do something like this:

```
call const_props(constituent_idx)%default_value(                           &
                                constituent_default_value,                 &
                                constituent_errflg,                        &
                                constituent_errmsg)
```

Just because that is how I usually see other long subroutine calls split-up elsewhere in CESM.

Reply:
Fixed!

Reply:
Based on our conversation today, I've reverted my fixes to manually format these lines and left them as the original commit and when the line length is increased in the PR for #210, that will re-format these with the updated line length to fix this formatting.